pvlib/tests/test_clearsky.py

```diff
@@ -214,8 +214,8 @@ def test_lookup_linke_turbidity_leapyear():
 
 
 def test_lookup_linke_turbidity_nointerp():
-    times = pd.date_range(start='2014-06-24', end='2014-06-25',
-                          freq='12h', tz='America/Phoenix')
+    times = pd.date_range(start='2014-06-01', end='2014-06-02',
```

Comment:
I'm still puzzled why, when `tz` is specified, `.tz_convert` is needed. Doesn't `...tz='America/Phoenix)` work?